src/main/java/org/homeschoolpebt/app/data/SentMessage.java

```diff
@@ -0,0 +1,44 @@
+package org.homeschoolpebt.app.data;
+
+import formflow.library.data.Submission;
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.util.Date;
+import java.util.UUID;
+
+import static jakarta.persistence.TemporalType.TIMESTAMP;
+
+@Entity
+@Table(name = "sent_messages")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class SentMessage {
+  @Id
+  @GeneratedValue
+  private UUID id;
+
+  @ManyToOne
+  @JoinColumn(name = "submission_id")
+  private Submission submission;
+
+  @Column(name = "message_name")
+  private String messageName;
+
+  @CreationTimestamp
+  @Temporal(TIMESTAMP)
+  @Column(name = "sent_at")
+  private Date sentAt;
+
+  @Column(name = "sent_status")
+  private String sentStatus;
```

Comment:
Yes - it's intended to eventually record "accepted" vs "delivered", which probably isn't necessary yet since the callbacks won't be implemented. Maybe I should take this out since it's not implemented yet. I will probably just merge this, though, and remove it later, since time is short.

Reply:
Out of curiosity, I'd be interested to hear about when any of the data in this table proves useful and how it is used. I'm assuming monitoring, troubleshooting, debugging, and other admin type activities but it would be nice to have some really specific examples.

Reply:
Sure, yeah, the hope is that by doing this recordkeeping, we can prevent ourselves later from sending duplicate messages. This will be helpful especially if anything goes wrong midway through the message sending code (so we can have an easy record of the people we successfully sent to), or if we want to change the logic later or something.